src/components/sharing/ShareBox/__tests__/ShareBox-test.js

```diff
@@ -0,0 +1,60 @@
+"use strict";
+
+jest.autoMockOff();
+
+describe('Share Box', function() {
+  var React                  = require('react/addons');
+  var ShareBox               = require('../');
+  var ShareIcon              = require('../../ShareIcon/');
+  var TestUtils              = React.addons.TestUtils;
+  var findByClass            = TestUtils.findRenderedDOMComponentWithClass;
+  var scryRenderedComponents = TestUtils.scryRenderedComponentsWithType;
+
+  var mockServices = [
+    { name: 'facebuck', url: 'http://facebuck.com/' },
+    { name: 'twattar', url: 'http://twatter.com/' },
+    { name: 'goobleplos', url: 'http://goobleplos.com/' }
+  ];
+
+  describe('component defaults', function() {
+    var shareBox;
+    var component;
+
+    beforeEach(function() {
+      shareBox  = <ShareBox services={ mockServices } />;
+      component = TestUtils.renderIntoDocument(shareBox);
+    });
+
+    it('renders something', function() {
+      expect(component).not.toBeNull();
+    });
+
+    it('will attempt to render service buttons', function() {
+      var result      = component.renderServiceButtons();
+      var iconWrapper = findByClass(component, 'ShareBox__icons');
+      var icons       = scryRenderedComponents(component, ShareIcon);
+
+      expect(result).toBeTruthy();
+      expect(icons.length).toBe(mockServices.length);
+    });
+  });
```

Comment:
Can you please add a test that clicking a service button opens share dialog. Will be much easier if you extract openDialog method into /src/lib/openDialog.js, and mock this file at the top, you can just test that a click event on a service icon calls the openDialog method with the expected params.